db-commons/src/main/scala/org/bitcoins/db/SQLiteUtil.scala

```diff
@@ -0,0 +1,12 @@
+package org.bitcoins.db
+
+object SQLiteUtil {
+
+  def backup(jdbcUrl: String, backupFileName: String): Unit = {
+    val conn = java.sql.DriverManager.getConnection(jdbcUrl)
+    try {
+      val _ = conn.createStatement().execute(s"BACKUP TO $backupFileName")
```

Comment:
might be worth doing some checks beforehand that the file doesn't exist already or that we have permissions to write there